x/oracle/types/keys.go

```diff
@@ -45,6 +46,18 @@ func GetOracleUpgradeKey(uniqueID string, address sdk.AccAddress) []byte {
 	return append(OracleUpgradeKey, CombineKeys([]byte(uniqueID), address)...)
 }
 
+func GetOracleUpgradeQueueKey(uniqueID string, addr sdk.AccAddress) []byte {
+	return append(OracleUpgradeQueueKey, CombineKeys([]byte(uniqueID), addr)...)
+}
+
+func GetOracleUpgradesKey(uniqueID string) []byte {
+	return append(OracleUpgradeQueueKey, []byte(uniqueID)...)
+}
```

Comment:
The name of ₩GetOracleUpgradesKey₩ seems to be the key to get `OracleUpgrade`. I suggest to rename it such as `GetOracleUpgradeQueueKeyByUniqueID`.
Or, you can use the `types.OracleUpgradeQueueKey` itself for iterator if you do not use the unique ID for this queue.

Reply:
Thanks, I use `types.OracleUpgradeQueueKey` itself for iterator.